editions/tw5.com/tiddlers/plugins/Plugin Ordering.tid

```diff
@@ -0,0 +1,51 @@
+created: 20220613115453346
+modified: 20220613171815188
+tags: PluginMechanism
+title: Plugin Ordering
+
+Using the Node.js client-server configuration, plugins are activated in the following order:
+
+# Plugins found using the OS environment variables
+#* And: [[PluginFolders]]
+#* See: [[Environment Variables on Node.js]]
+
+# Plugins stored in the wiki `/plugin` path
+#* See: [[PluginFolders]]
+
+# Plugins specified in the command line
+#* See: [[Using TiddlyWiki on Node.js]] ... the notes about <<.from-version "5.1.20">>
+
+# Plugins imported by drag & drop as wiki content
+#* See: [[Installing a plugin from the plugin library]] or [[Manually installing a plugin]]
+
+''Important:''
+
+<<<
+''Elements lower in the list take precedence''.
+
+Including a plugin as an ordinary tiddler by ''drag & drop''  into the browser, will result in the plugin only being active in the browser. It will not be available under Node.js!
```

Comment:
Drag and drop shouldn't be in bold; and we should spell "and" in full. It's important to be consistent with the existing docs.

Also the exlamation mark is not necessary here.